patch/kconfig-exclusions

```diff
@@ -15,6 +15,10 @@ CONFIG_THERMAL_STATISTICS
 
 [arm64]
 
+[marvell-arm64]
+CONFIG_UBIFS_FS_ZSTD
+CONFIG_CRYPTO_DEV_SAFEXCEL
```

Comment:
Any reason these two kernel configs are being unset here? Is it just that they're not used, or that they break something? If the former, I'd like to keep them enabled.

Reply:
@saiarcot895 Below are the reasons for exclusions. Same information is updated in commit summary.

Overcome following dmesg error during boot.
    - UBIFS error: cannot initialize compressor zstd, error -2
        - CONFIG_UBIFS_FS_ZSTD

    - crypto-safexcel f2800000.crypto: HW init failed (-2)
        - CONFIG_CRYPTO_DEV_SAFEXCEL
        - EIP197 cryptographic engine is not used.